### PR TITLE
Add altitude and accuracy to capture location submissions

### LINKED ISSUE #1329: [Submission details] Display elevation and accuracy for capture location
Right now capture location tasks are treated the same as a point. Although they have some additional data (elevation and accuracy), but it is not shown anywhere.

### PR DESCRIPTION
**Pull request by amysorto**
Fixes #1329 

Screenshot 2024-10-09 at 10 58 29 AM

### CODE REVIEW

**Review by gino-m — APPROVED** [review]
Nice work! 🧭

File: web/src/app/pages/main-page-container/main-page/secondary-side-panel/submission-panel/submission-panel.component.ts [hunk 1]
```diff
@@ -109,10 +109,20 @@ export class SubmissionPanelComponent implements OnInit, OnDestroy {
...
     // x represents longitude, y represents latitude
-    const {x, y} = (this.getTaskSubmissionResult(task)!.value as Point).coord;
+    const {coord, accuracy, altitude} = this.getTaskSubmissionResult(task)!
+      .value as Point;
+    const {x, y} = coord;
     const long = Math.abs(x).toString() + (x > 0 ? '° E' : '° W');
     const lat = Math.abs(y).toString() + (y > 0 ? '° N' : '° S');
-    return lat + ', ' + long;
+
+    let result = lat + ', ' + long;
```
Comment (gino-m) [inline web/src/app/pages/main-page-container/main-page/secondary-side-panel/submission-panel/submission-panel.component.ts:118]: Consider using string templates here and below instead

File: web/src/app/converters/submission-data-converter.ts [hunk 2]
```diff
@@ -107,7 +107,15 @@ function taskDataPbToModelValue(
   } else if (drawGeometryResult)
     return geometryPbToModel(drawGeometryResult.geometry!) as Polygon;
   else if (captureLocationResult)
-    return new Point(coordinatesPbToModel(captureLocationResult.coordinates!));
+    return new Point(
+      coordinatesPbToModel(captureLocationResult.coordinates!),
+      captureLocationResult.accuracy === null
```
Comment (gino-m) [inline web/src/app/converters/submission-data-converter.ts:112]: Does the elvis operator help here? `captureLocationResult.accuracy ?: null`?

File: web/src/app/converters/submission-data-converter.ts [hunk 3]
```diff
@@ -107,7 +107,15 @@ function taskDataPbToModelValue(
...
+        ? undefined
+        : captureLocationResult.accuracy,
+      captureLocationResult.altitude === null
```
Comment (gino-m) [inline web/src/app/converters/submission-data-converter.ts:115]: Same here?

**Review by rfontanarosa — APPROVED** [review]

File: web/src/app/pages/main-page-container/main-page/secondary-side-panel/submission-panel/submission-panel.component.ts (hunk 1 above)
Comment (rfontanarosa) [reply web/src/app/pages/main-page-container/main-page/secondary-side-panel/submission-panel/submission-panel.component.ts:118]: see https://github.com/google/ground-platform/pull/2102

File: web/src/app/converters/submission-data-converter.ts (hunk 2 above)
Comment (rfontanarosa) [reply web/src/app/converters/submission-data-converter.ts:112]: see https://github.com/google/ground-platform/pull/2102

File: web/src/app/converters/submission-data-converter.ts (hunk 3 above)
Comment (rfontanarosa) [reply web/src/app/converters/submission-data-converter.ts:115]: see https://github.com/google/ground-platform/pull/2102

**Pull request merged**